snuba/clickhouse.py

```diff
@@ -375,6 +376,21 @@ def for_schema(self):
         return ', '.join(column.for_schema() for column in self.columns)
 
 
+class TableSchema(object):
+    """
+    Represents the full set of columns in a clickhouse table, this only contains
+    basic metadata for now. The code to generate the schema comes in a followup PR.
+    """
+
+    def __init__(self):
+        self.CLICKHOUSE_CLUSTER = None
+        self.DATABASE = None
+        self.LOCAL_TABLE = None
+        self.DIST_TABLE = None
+        self.QUERY_TABLE = None
```

Comment:
As a followup I would like to kill this constant. We should just have two table constants:
1) the table you refer to insert/select, which will be the distributed one in prod and the local one in dev
2) the local table for the few cases where we specifically need to access it in prod.